cmd/axon/main.go

```diff
@@ -59,6 +59,11 @@ func main() {
 		logger.WithError(err).Fatal("unable to connect to target database")
 	}
 
+	err = checkTargetVersion(targetDBConn)
+	if err != nil {
+		logger.WithError(err).Fatal("unable to use to target database")
```

Comment:
This message should mention that check target version failed?

Reply:
Added